src/client/ws/packetHandler.js

```diff
@@ -1,48 +1,44 @@
-function handle(client, { event, args }, id) {
-    function getServer(id) {
```

Comment:
Why was the `getServer()` function removed here?

Reply:
You don't have servers in cache if you only use sockets. Accessing cache at every socket event is inefficient, imagine every server console message on busy game or web servers